src/Compilers/Core/Portable/Emit/EmitOptions.cs

```diff
@@ -104,6 +105,12 @@ public sealed class EmitOptions : IEquatable<EmitOptions>
         /// </summary>
         public string? RuntimeMetadataVersion { get; private set; }
 
+        /// <summary>
+        /// The default source encoding used to parse source before producing a compilation. Only embedded in
+        /// the pdb if 
```

Comment:
>if [](start = 20, length = 2)

incomplete

Reply:
that's what I git for committing late at night >.>